.github/workflows/linters.yml

```diff
@@ -0,0 +1,34 @@
+name: Run Linters
```

Comment:
Are we using GH workflows?

Reply:
Thanks @awels - removed the github action file.